### ISSUE TITLE
Benefits section needs content changes and better screen reader support

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#110]
- [x] update info:
- Flexible work models: at the office, hybrid, or remote; no strict work hours
- Mouthwatering perks: leave everything the same, just change the last one to "Friday bars"
- Wellness: plenty of space for games and relaxation; small gym
- Support in education: leave the same
- Gifts and bonuses: leave the same
- Support for professional growth: leave the same

- [x] aria-label for not visible label of each list
- [ ] for later: group all lists in one list (with a label)
